extension/encoding/jsonlogencodingextension/config.go

```diff
@@ -7,24 +7,38 @@ import "fmt"
 
 type JSONEncodingMode string
 
+type ProcessingMode string
+
 const (
 	JSONEncodingModeBodyWithInlineAttributes JSONEncodingMode = "body_with_inline_attributes"
 	JSONEncodingModeBody                     JSONEncodingMode = "body"
+	ArrayMode                                ProcessingMode   = "array"
+	SingleMode                               ProcessingMode   = "single"
+	NDJsonMode                               ProcessingMode   = "ndjson"
```

Comment:
```suggestion
	NDJSONMode                               ProcessingMode   = "ndjson"
```

Nit: capitalise according to https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
Fixed this in the latest commit :)

Reply:
This is no longer used so will resolve :)